### PR TITLE
Enable Horizon

### PR DESCRIPTION
**Pull request by eugenegrebeshkov**

### CODE REVIEW

File: custom-images/docker-compose.override.yml [hunk 1]
```diff
@@ -110,4 +114,38 @@ services:
...
+      Sitecore_SitecorePlatform__ContentManagementInternalUrl: http://cm
+      Sitecore_Sitecore__Authentication__OpenIdConnectOptions__RequireHttpsMetadata: 'false'
+      Sitecore_Sitecore__Authentication__OpenIdConnectOptions__Authority: https://${ID_HOST}
+      Sitecore_Sitecore__Authentication__OpenIdConnectOptions__CallbackAuthority: https://${AH_HOST}
+      Sitecore_Sitecore__Authentication__OpenIdConnectOptions__InternalAuthority: http://id
+      Sitecore_Sitecore__Authentication__BearerAuthenticationOptions__Authority: https://${ID_HOST}
+      Sitecore_Sitecore__Authentication__BearerAuthenticationOptions__InternalAuthority: http://id
+      Sitecore_Sitecore__Authentication__BearerAuthenticationOptions__RequireHttpsMetadata: 'false'
+      Sitecore_Plugins__Filters__ExperienceAccelerator: +SXA
+      Sitecore_Plugins__Filters__ContentHub: +ContentHub
```
Comment (stasmaxymov) [inline custom-images/docker-compose.override.yml:137]: Let's disable contenthub, as there are no ContentHub module in this deployment

Comment (eugenegrebeshkov) [reply custom-images/docker-compose.override.yml]: Done

File: custom-images/.env [hunk 2]
```diff
@@ -12,6 +12,7 @@ LOCAL_DATA_PATH=.\docker\data
 CD_HOST=cd.dockerexamples.localhost
 CM_HOST=cm.dockerexamples.localhost
 ID_HOST=id.dockerexamples.localhost
+AH_HOST=ah.dockerexamples.localhost
```
Comment (stasmaxymov) [inline custom-images/.env:15]: Let's name it as HRZ_HOST.  Authoring host is the internal name.

Comment (eugenegrebeshkov) [reply custom-images/.env]: Done

**Review by stasmaxymov — DISMISSED** [review]

**Review by nickwesselman — APPROVED** [review]

**Pull request merged**